nvflare/lighter/impl/static_file.py

```diff
@@ -127,9 +130,9 @@ def _build_server(self, server, ctx):
         ctx["admin_port"] = admin_port
         fed_learn_port = server.props.get("fed_learn_port", 8002)
         ctx["fed_learn_port"] = fed_learn_port
-        ctx["server_name"] = server.name
-        server_0["service"]["target"] = f"{server.name}:{fed_learn_port}"
-        server_0["admin_host"] = server.name
+        ctx["server_name"] = self.get_server_name(server)
+        server_0["service"]["target"] = f"{self.get_server_name(server)}:{fed_learn_port}"
+        server_0["admin_host"] = self.get_server_name(server)
```

Comment:
instead of wrapping "server.name" into a method,

suggest just adding type hint `server: Participant`